README.md

```diff
@@ -25,12 +25,22 @@ yarn add @fullstory/server-api-client
 
 #### Initializing the Client
 
-Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/).
+Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/). The `FullStoryOptions` used in `init` function will be used for all server API requests, unless overridden by [per request options](#request-options).
 
 ```ts
 import { init } from '@fullstory/server-api-client';
 
-const fsClient = init({ apiKey: '<YOUR_API_KEY>' });
+const fsOpts: FullStoryOptions = {
+  // apiKey is required to init the FullStory client
+  apiKey: '<YOUR_API_KEY>',
+  
+  // integrationSource is generally intended for FullStory developers
+  // or FullStory partners while building integrations.
+  //Generally it should be left empty.
```

Comment:
This is a good doc string to include in the source code too - so that it shows up in IDEs

Reply:
Can up update this to include guidance around FullStory developers and Partners: https://github.com/fullstorydev/fullstory-node-sdk/blob/9788d9cf9a0bbb7b11bb0835e32294fc827d9414/src/http/options.ts#L5